Gemfile.lock

```diff
@@ -499,38 +503,38 @@ GEM
     omniauth-rails_csrf_protection (1.0.1)
       actionpack (>= 4.2)
       omniauth (~> 2.0)
-    pagy (3.6.0)
-    parallel (1.19.2)
-    parallel_tests (2.21.1)
+    pagy (3.14.0)
+    parallel (1.24.0)
+    parallel_tests (2.32.0)
       parallel
     parser (2.7.2.0)
       ast (~> 2.4.1)
-    path_expander (1.0.4)
-    pg (1.1.3)
-    pry (0.10.3)
+    path_expander (1.1.1)
+    pg (1.5.4)
+    pry (0.12.2)
       coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
+      method_source (~> 0.9.0)
     pry-byebug (3.3.0)
       byebug (~> 8.0)
       pry (~> 0.10)
-    pry-rails (0.3.5)
-      pry (>= 0.9.10)
-    pry-rescue (1.4.5)
+    pry-rails (0.3.9)
+      pry (>= 0.10.4)
+    pry-rescue (1.6.0)
       interception (>= 0.5)
-      pry
+      pry (>= 0.12.0)
     pry-stack_explorer (0.4.9.3)
       binding_of_caller (>= 0.7)
       pry (>= 0.9.11)
-    public_suffix (5.0.1)
+    public_suffix (5.0.4)
     puma (5.6.8)
       nio4r (~> 2.0)
     pyu-ruby-sasl (0.0.3.3)
     racc (1.7.3)
     rack (2.2.8)
-    rack-mini-profiler (1.1.4)
+    rack-mini-profiler (1.1.6)
       rack (>= 1.2.0)
-    rack-protection (3.1.0)
+    rack-protection (3.2.0)
```

Comment:
### [rack-protection](http://sinatrarb.com/protection/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.1.0 → 3.2.0

[_~~change-log~~, [source-code](https://github.com/sinatra/sinatra/tree/main/rack-protection), [gem-diff](https://my.diffend.io/gems/rack-protection/3.1.0/3.2.0)_]





<details>
<summary>Commits</summary>

A change of **36** commits. See the full changes on [the compare page](https://github.com/sinatra/sinatra/compare/v3.1.0...v3.2.0).

These are the first 10 commits:
- (35bc85b) [Fix PR link in changelog (#1936)](https://github.com/sinatra/sinatra/commit/35bc85b18a4154a5bf94044460c992b58d65c964)
- (0889424) [feature: add `except` to `IndifferentHash` (#1940)](https://github.com/sinatra/sinatra/commit/0889424a43a18d7a87fc4a51ec7a2082b51348a0)
- (4f5ca85) [Update SECURITY.md (#1941)](https://github.com/sinatra/sinatra/commit/4f5ca85d408b2a64d9a0245eb5a1a14eaa2366b2)
- (ecee206) [Use zzak/action-discord@v6 (#1942)](https://github.com/sinatra/sinatra/commit/ecee206883daace7ff7cf9090c6fca80da792945)
- (9c08af4) [Use relative URL for the 404 image (#1943)](https://github.com/sinatra/sinatra/commit/9c08af41c97ddf557e2f4c1267c6840b3c756cc0)
- (23a532a) [Update link for slim template (#1944)](https://github.com/sinatra/sinatra/commit/23a532a92d6afb549c309e792b37be95c62ab4b7)
- (e130771) [Add `base64` to rack-protection runtime dependencies (#1946)](https://github.com/sinatra/sinatra/commit/e130771db80acc380f05323fc7d7c0fa258b7d42)
- (5b38ee1) [Manage development dependencies with `Gemfile`s](https://github.com/sinatra/sinatra/commit/5b38ee1df438eab76298a79ac4c85e68a59cd22f)
- (9417bd4) [Remove unnecessary sinatra-contrib dev deps](https://github.com/sinatra/sinatra/commit/9417bd48ee4f73a3941cee8fa6e03d76208abb8b)
- (3363cc7) [Remove Gemfile groups, it is all test/dev deps](https://github.com/sinatra/sinatra/commit/3363cc7b1612a24c07caddc94a348de52131e45e)


</details>